Router.svelte

```diff
@@ -579,9 +603,6 @@ const unsubscribeLoc = loc.subscribe(async (newLoc) => {
             componentParams = null
         }
 
-        // Set static props, if any
```

Comment:
I think the new code should have been here. This is because there is now async code (which could do things like making network requests, which take time) which is now delaying showing the "loading" component too. That would damage the user experience.